src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -175,6 +175,11 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'type': 'boolean',
 		'description': nls.localize('closeOnFileDelete', "Controls if editors showing a file should close automatically when the file is deleted or renamed by some other process. Disabling this will keep the editor open as dirty on such an event. Note that deleting from within the application will always close the editor and that dirty files will never close to preserve your data."),
 		'default': true
+	},
+	'workbench.fontAliasing': {
```

Comment:
As i am just learning the code base -- is plaform dependent configuration options common? This has one side effect when using settings-sync to syncronize configuration between mac and windows machines it'll mark "unknown" configuration properties as invalid on another platform.